packages/elements/src/internals/machines/third-party/third-party.actors.ts

```diff
@@ -78,10 +78,6 @@ export const handleRedirectCallback = fromCallback<AnyEventObject, HandleRedirec
 
     void loadedClerk.handleRedirectCallback(
       {
-        signInForceRedirectUrl: ClerkJSNavigationEvent.complete,
-        signInFallbackRedirectUrl: ClerkJSNavigationEvent.complete,
-        signUpForceRedirectUrl: ClerkJSNavigationEvent.complete,
-        signUpFallbackRedirectUrl: ClerkJSNavigationEvent.complete,
```

Comment:
Now that we're not passing `beforeEmit` to `setActive` internally, our custom navigate hackery wasn't working as intended here. Removing the URLs allows navigate to complete.

Reply:
Does this mean we broke the usage of `handleRedirectCallback` for everyone ?

Reply:
@panteliselef for elements, yes 😅

Reply:
Hey, trying to use <SignIn.Captcha /> but I see `Property 'Captcha' does not exist ...`.

I believe you forgot to export this component in `index.ts` right?

Reply:
@corentinmusard yes, good catch. PR here https://github.com/clerk/javascript/pull/4548